build: npm package for v1.1.14-v1.1.16 contains .git folder

<!-- 
Filling out this template is required! Do not delete it when submitting your issue! Without this information, your issue may be auto-closed. 

Please submit questions to the [AngularJS Material Forum](https://groups.google.com/forum/#!forum/ngmaterial) instead of submitting an issue.
-->
## Bug, enhancement request, or proposal:
Bug

<!-- Please note that we are not accepting major feature requests, i.e. requests for new components, at this time. -->

#### Detailed Reproduction Steps: 
1. npm install angular-material@1.1.14
2. check the node_modules\angular-material directory

### What is the expected behavior?
there is no .git folder

### What is the current behavior?
there is a .git folder

### What is the use-case or motivation for changing an existing behavior?
npm doesn't like having sub modules or repos in the node_modules folder, and complains with the following error

npm ERR! git {myproject}\node_modules\angular-material: Appears to be a git repo or submodule.
npm ERR! git     {myproject}\node_modules\angular-material
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

### Which versions of AngularJS, Material, OS, and browsers are affected?

- AngularJS Material: 1.1.14
 
### Is there anything else we should know? Stack Traces, Screenshots, etc.


<!-- Please double check that you have provided the required reproduction steps and a Demo via CodePen, Plunker, or GitHub repo. -->

--- a/release.js
+++ b/release.js
@@ -251,6 +251,7 @@
       'git pull --rebase --strategy=ours',
       'git push',
       'git push --tags',
+      'rm -rf .git/',
       'npm publish',
       'cd ..'
     );

